src/test/input_byte_size_test.cc

```diff
@@ -258,19 +258,20 @@ char InputByteSizeTest::input_data_string_
 
 TEST_F(InputByteSizeTest, ValidInputByteSize)
 {
+  const char* model_name = "savedmodel_zero_1_float32";
```

Comment:
I think we are now using the same model for all our tests below. Can we add model_name_ as class member in L245-252? This way we can use the model_name_ everywhere here and below.
It also shows our intention using the same model.

Reply:
Not really. String input tests use savedmodel_zero_1_object instead.